formatters/html/html.go

```diff
@@ -64,24 +75,65 @@ func BaseLineNumber(n int) Option {
 func New(options ...Option) *Formatter {
 	f := &Formatter{
 		baseLineNumber: 1,
+		preWrapper:     defaultPreWrapper,
 	}
 	for _, option := range options {
 		option(f)
 	}
 	return f
 }
 
+// PreWrapper defines the operations supported in WithPreWrapper.
+type PreWrapper interface {
+	// WriteStart is called to write a start <pre> element.
+	// The code flag tells whether this block surrounds
+	// highlighted code. This will be false when surrounding
+	// line numbers.
+	WriteStart(w io.Writer, code bool, styleAttr string)
```

Comment:
Can you change these methods to just return a string rather than take a writer? And rename them to Start and End?

Reply:
Will do.